src/SAVE/aorsa2dMain.F

```diff
@@ -3405,10 +3414,13 @@ program aorsa2dMain
      .                               - z_slo * xna_slo(i, j)   ) / z1
      
                if (xn1a(i, j) .le. 0.0) then
-	          xn1a(i, j) = 1.0e-10
 		  if(myid .eq.0) write(6, *) xnea(i,j), xn1a(i,j), 
      .               xn2a(i,j), xn3a(i,j), xn4a(i,j), xn5a(i,j), 
      .               xn6a(i,j), xna_slo(i,j)
+	          xn1a(i, j) = 1.0e-10
```

Comment:
Where does this exit to afterwards? If the program continues (i.e., exits to outside a for loop it's presently in), perhaps adjust this write statement to say what it is doing, e.g., "Warning - xn1a <= 0, setting to 1e-10" or some such. Or does this just exit the program as a whole?